Buildenator/Generators/PropertiesStringGenerator.cs

```diff
@@ -197,10 +267,100 @@ private string GenerateAddToMethodDefinition(ITypedSymbol typedSymbol)
                 ? new System.Collections.Generic.List<{elementTypeName}>({fieldName}.Value.Object) 
                 : new System.Collections.Generic.List<{elementTypeName}>();
             list.AddRange(items);
-            {fieldName} = new {DefaultConstants.NullBox}<{typedSymbol.TypeFullName}>((({typedSymbol.TypeFullName})list));
+            {fieldName} = new {DefaultConstants.NullBox}<{typedSymbol.TypeFullName}>(({typedSymbol.TypeFullName})list);
             return this;
         }}";
 	}
 
 	private string CreateAddToMethodName(ITypedSymbol property) => $"AddTo{property.SymbolPascalName}";
+
+	/// <summary>
+	/// Tries to get the builder name for a property's type.
+	/// Returns true if a child builder exists for the property's type.
+	/// </summary>
+	private bool TryGetChildBuilderName(ITypedSymbol typedSymbol, out string childBuilderName)
+	{
+		childBuilderName = string.Empty;
+		if (_entityToBuilderMappings == null)
+			return false;
+		
+		var typeFullName = typedSymbol.TypeFullName;
+		return _entityToBuilderMappings.TryGetValue(typeFullName, out childBuilderName!);
```

Comment:
Do not use bang operators!!
Instead of bool rather return nullable childBuilderName